Dockerfile

```diff
@@ -25,7 +25,7 @@ FROM alpine:3.10 AS app
 WORKDIR /app
 
 COPY --from=builder /src/opam-install/bin/bot bot.exe
-COPY --from=builder /src/make_ancestor.sh /src/coq_bug_minimizer.sh ./
+COPY --from=builder /src/make_ancestor.sh /src/coq_bug_minimizer.sh /src/coqbot.toml ./
```

Comment:
What is this `/src/coqbot.toml` file? The only one that is defined is `coqbot.toml`, no? And it should be on a layer on top. Maybe this is just because I'm looking at an intermediate step here.

Reply:
That's right, this line simplifies to: `COPY make_ancestor.sh coq_bug_minimizer.sh coqbot.toml ./`. There's no need to copy those files from the `builder` stage when it's already accessible.

Reply:
Interesting simplification. But this wasn't my main point. My main point was that I would have expected the main `Dockerfile` (the one that is going to get published on GitHub Packages) to not contain any `coqbot.toml` and another `Dockerfile` to be built on top with this configuration file for deployment to Heroku (exactly as documented in the README in fact).

Reply:
Oh yeah, this is much cleaner.

Reply:
We will need to adapt are current GitHub workflow to deploy to Heroku if we create our docker image from an unpublished release docker image (which must be built beforehand).